SR2023/2022-09-29-confirm-team-details.md

```diff
@@ -0,0 +1,24 @@
+---
+to: SR2023 sign-ups
+subject: Confirming your details for SR2023
+---
+
+Hello!
+
+Thank you for your interest in Student Robotics 2023.
+
+Before we can confirm your place, we have a few questions and extra pieces of information we need.
+
+1. Can you confirm you are over the age of 18, and officially related to the institution you are bringing a team from (eg teacher at a school, leader at a scout group etc).
```

Comment:
Is the latter part of this relevant? I would have thought we care more along the lines of "and will have permission to bring the competitors to events/act as a guardian for them at our events"?

Reply:
I think it is, yes. For example, a parent and a bunch of students from a school don't really have permission to represent themselves as being a team from the school.

Reply:
Yeah, that's kinda why I was thinking the wording needs a tweak? Not sure what you're saying here.